### DISCUSSION TITLE
Upgrade to .NET 7

### DISCUSSION THREAD

**Discussion by SirJohnK** [SirJohnK/LocalizationResourceManager.Maui#13] [Announcements]
Since .NET MAUI 6 is out of support, this library will soon upgrade to .NET MAUI 7 (V1.1.0)! 😀
[.NET Multi-platform App UI (.NET MAUI) Support Policy](https://dotnet.microsoft.com/en-us/platform/support/policy/maui)

**Comment by ramonB1996** [comment]
.NET8 is coming up in 2 months as well. Will this library upgrade to it soon after?

I really like the ease of use, thank you for your work!

**Reply by SirJohnK** [reply]
Thanks!😀 Upgrade to .NET 8, will only be done if necessary. The .NET 7 version should work just fine with .NET 8.👍 Planned uppgrade to .NET 8 is when .NET MAUI 7 goes out of support in May 2024.😅

**Comment by SirJohnK** [comment]
[V1.1.0](https://github.com/SirJohnK/LocalizationResourceManager.Maui/releases/tag/1.1.0) targeting .NET 7 is now released!
